osu.Game/Overlays/Profile/ProfileHeader.cs

```diff
@@ -0,0 +1,463 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Users;
+
+namespace osu.Game.Overlays.Profile
+{
+    public class ProfileHeader : Container
+    {
+        private readonly OsuTextFlowContainer infoTextLeft, infoTextRight;
+        private readonly FillFlowContainer<SpriteText> scoreText, scoreNumberText;
+
+        private readonly Container coverContainer, chartContainer, supporterTag;
+        private readonly Sprite levelBadge;
+        private readonly SpriteText levelText;
+        private readonly GradeBadge gradeSSPlus, gradeSS, gradeSPlus, gradeS, gradeA;
+        private readonly Box colourBar;
+
+        private const float cover_height = 350, info_height = 150, info_width = 220, avatar_size = 110, level_position = 30, level_height = 60;

```

Comment:
Not sure if this is a thing we regulate but I usually prefer constants at the top of the class.